src/contexts/Graph.js

```diff
@@ -9,13 +9,17 @@ export const GraphContext = createContext()
 
 //ApolloClients
 const clientMainnet = new ApolloClient({
-  uri: 'https://api.thegraph.com/subgraphs/name/tellor-io/tellorxoraclemainhgraph',
+  uri: 'https://api.thegraph.com/subgraphs/name/raynharr/tellor-flex-ethmain-graph',
```

Comment:
Supposed to be your name in the URI?

Reply:
For now since I deployed the graph from my account. Def should get set up under Tellor account down the road if Graph is long-term solution.